akka-docs/src/main/paradox/typed/extending.md

```diff
@@ -35,10 +35,10 @@ Scala
 Java
 :  @@snip [ExtensionDocTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/extensions/ExtensionDocTest.java) { #extension }
 
-This is the public API of your extension. Internally in this example we instantiate our expensive database connection.
-The `DatabaseConnectionPool` can be looked up this way any number of times and it will return the same instance. 
+This is the public API of your extension. Internally in this example we instantiate our expensive database connection. 
 
 Then create an @apidoc[akka.actor.typed.ExtensionId] to identify the extension.
+@scala[A good convention is to let the companion object of the `Extension` be the `ExtensionId`.]@java[A good convention is to define the `ExtensionId` as a static inner class of the `Extension`.]
```

Comment:
"called `Id`" even?

Reply:
changed it to: A good convention is to define the `Id extends ExtensionId` as a static inner class of the `Extension`

was difficult to make that sentence parsable with something like:
A good convention is to define the `ExtensionId`, called `Id`, as a static inner class of the `Extension`

Reply:
👍